### ISSUE TITLE
The `SequentialLR` scheduler uses a deprecated pattern 

### ISSUE THREAD

**Issue by OliverFM** [pytorch/pytorch#67958]
## To Reproduce

1. run `python test/test_optim.py`

This produces several warnings. If one runs on the branch of PR #67954, only one warning is produced:
```
.........................................................................................../home/oliver/code/playground/improve-optim-tests/torch/optim/lr_scheduler.py:154: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)
 ```
 
 This comes from the tests of `SequentialLR` as it calls `scheduler.step(0)` on [line 637](https://github.com/pytorch/pytorch/blob/ae501a9727c16b8bc83c2fd7b40d9d801133ac91/torch/optim/lr_scheduler.py#L632).

## Expected behavior

Code ought to work without producing warnings.

## Environment
```
❯ cat torch_env.txt
Collecting environment information...
PyTorch version: 1.11.0.dev20211105
Is debug build: False
CUDA used to build PyTorch: 11.3
ROCM used to build PyTorch: N/A

OS: Ubuntu 18.04.6 LTS (x86_64)
GCC version: (Ubuntu 9.4.0-1ubuntu1~18.04) 9.4.0
Clang version: Could not collect
CMake version: version 3.19.6
Libc version: glibc-2.27

Python version: 3.9.7 (default, Sep 16 2021, 13:09:58)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-5.4.0-87-generic-x86_64-with-glibc2.27
Is CUDA available: True
CUDA runtime version: Could not collect
GPU models and configuration:
GPU 0: GeForce GTX 1080 Ti
GPU 1: GeForce GTX 1080 Ti
GPU 2: GeForce GTX 1080 Ti
GPU 3: GeForce GTX 1080 Ti
GPU 4: GeForce GTX 1080 Ti
GPU 5: GeForce GTX 1080 Ti
GPU 6: GeForce GTX 1080 Ti
GPU 7: GeForce GTX 1080 Ti

Nvidia driver version: 450.102.04
cuDNN version: Probably one of the following:
/usr/lib/x86_64-linux-gnu/libcudnn.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_adv_infer.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_adv_train.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_infer.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_train.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_ops_infer.so.8.0.5
/usr/lib/x86_64-linux-gnu/libcudnn_ops_train.so.8.0.5
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] mypy==0.910
[pip3] mypy-extensions==0.4.3
[pip3] numpy==1.21.2
[conda] blas                      1.0                         mkl
[conda] cudatoolkit               11.3.1               h2bc3f7f_2
[conda] mkl                       2021.4.0           h06a4308_640
[conda] mkl-include               2021.4.0           h06a4308_640
[conda] mkl-service               2.4.0            py39h7f8727e_0
[conda] mkl_fft                   1.3.1            py39hd3c417c_0
[conda] mkl_random                1.2.2            py39h51133e4_0
[conda] mypy                      0.910              pyhd3eb1b0_0
[conda] mypy_extensions           0.4.3            py39h06a4308_0
[conda] numpy                     1.21.2           py39h20f2e39_0
[conda] numpy-base                1.21.2           py39h79a1101_0
[conda] pytorch-mutex             1.0                        cuda    pytorch-nightly
```
## Additional context

I found this issue while I was working on #67954 which in turn fixes #67696.
I'm not sure why this pattern is still used -- but I haven't yet tried to figure out the details of this. If I get the time I am tempted to fix this.

**Comment by zeshengzong** [comment]
Hi, this already fixed in #149392 and this issue can be closed, thanks!
